### PR TITLE
Ready for EC2

### PR DESCRIPTION
**Pull request by JasperCDev**
@jayrbarr @timothydadams

### CODE REVIEW

File: newrelic.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['SDC-Jasper-Service'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'c73b804e50b6ee3984dfb686909caf7730ecNRAL',
+  logging: {
```
Comment (jayrbarr) [inline newrelic.js]: Ah, you posted the license key too! I'm not sure if it's a huge deal or not, but you will want to at least .gitignore and rm --cached

**Review by jayrbarr — APPROVED** [review]
lgtm

File: newrelic.js (hunk 1 above)
Comment (JasperCDev) [reply newrelic.js]: thank you Jay!

File: stressTest.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      x-my-service-auth: '987401838271002188298567'
+scenarios:
+  - flow:
+    - loop:
+      - function: 'randomId'
+      - get:
+          url: "/media/?id={{id}}"
+      count: 100
+    - loop:
+      - post:
```
Comment (timothydadams) [inline stressTest.yml:18]: is it actually posting anything to the db?

Comment (JasperCDev) [reply stressTest.yml:18]: yes it is, its just generic data that I declare in the api endpoint function.

**Pull request merged**